srcpkgs/openttd/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'openttd'
 pkgname=openttd
-version=1.10.3
+version=12.2
 revision=4
 _gfxver=0.6.0
 _sfxver=0.2.3
```

Comment:
there are new versions of [opengfx](https://www.openttd.org/downloads/opengfx-releases/latest.html) and [opensfx](https://www.openttd.org/downloads/opensfx-releases/latest) available, too